askbot/admin.py

```diff
@@ -141,8 +141,8 @@ def is_published(self, obj):
 
 class PostRevisionAdmin(admin.ModelAdmin):
     list_display = ('id', 'post_id', 'thread_name', 'revision', 'revised_at', 'author', 'approved')
-    list_filter = ('approved',)
-    search_fields = ('post__id',)
+    list_filter = ('approved', 'author')
```

Comment:
Good point - our instance has a manageable number of users at the moment, but that won't apply to all askbot instances - thanks for the catch! This pull request has already been merged, so see fix at https://github.com/ASKBOT/askbot-devel/pull/430.